src/stories/pages/ExperimentsPage/ExperimentsPage.js

```diff
@@ -38,92 +28,87 @@ export const UserContext = React.createContext();
  * )
  */
 export const ExperimentsPage = () => {
-  const [logInState, setLogInState] = useState({ isLoggedIn: true });
-  const [allExperiments, setAllExperiments] = useState(PLACEHOLDER_EXPERIMENTS);
+  const { user } = useAuth();
+  const [ongoingExperiments, setOngoingExperiments] =
+    useState(PLACEHOLDER_EXPERIMENTS);
+  const [inactiveExperiments, setInactiveExperiments] =
+    useState(PLACEHOLDER_EXPERIMENTS);
   const [error, setError] = useState();
 
-  // TODO #1.3(fernandowinfield): remove the service below.
-  const userService = UserServices.getInstance();
-
-  // TODO #1.4(fernandowinfield): adjust `useEffect()` implementation below to
-  // use new auth context.
-  useEffect(() => {
-    userService.getCurrentUser(setLogInState).catch((e) => {
-      setLogInState({ isLoggedIn: false });
-      setError(e);
-    });
-  }, [logInState.isLoggedIn]);
-
   useEffect(() => {
     getAllExperiments().then((response) => {
       if (response.status === 200) {
-        // TODO #3.1(fernandowinfield): Filter ongoing and inactive experiments
-        // and set state for both instead of setting state for all.
-        setAllExperiments(response.data);
+        const ongoingExperiments = response.data.filter((experiment) =>
+          experiment.isOngoing);
+        const inactiveExperiments = response.data.filter((experiment) =>
+          !experiment.isOngoing);
+        setOngoingExperiments(ongoingExperiments);
+        setInactiveExperiments(inactiveExperiments);
       } else {
         setError(response.error);
       }
     });
   }, []);
 
-  // TODO #3.2(fernandowinfield): Map ongoingExperiments instead of all.
-  const ongoingExperimentsSlides = allExperiments ?
-    allExperiments.map(
-      (
-        // TODO #2.1(fernandowinfield): add/remove fields below as needed.
-        { title = 'Title of Experiment',
-          description = 'Short description of experiment',
-          opened = 'mm-dd-yyyy',
-          admin = 'Admin name',
-          researchers = ['Researcher Name'],
-        },
-        i,
-      ) =>
+  // Generate slider's slides for ongoing experiments (if any)
+  const ongoingExperimentsSlides = (ongoingExperiments &&
+    ongoingExperiments.length > 0) ?
+    ongoingExperiments.map((exp, i) => {
+      const experimentInfo = {
+        title: exp.title,
+        body: exp.description.length > MAX_DESCRIPTION_LENGTH ?
```

Comment:
I think let's just use 'body: exp.description' here. Trimming it doesn't seem to work well